client/client.go

```diff
@@ -80,17 +79,17 @@ func New(ctx context.Context, logger zerolog.Logger, s *Spec) (Client, error) {
 	}
 
 	if appID == 0 {
```

Comment:
Edge case, nitpick: We don't actually know how GitHub generates their app IDs. It's possible that you could have a legitimate app ID of 0. I think parsing this as a string, where the nil string is `""` is safer